packages/flutter/test/material/icon_button_test.dart

```diff
@@ -75,6 +75,39 @@ void main() {
     expect(iconButton.size, const Size(70.0, 70.0));
   });
 
+  testWidgets('Small icons with non-null constraints can be <48dp', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      wrap(
+        child: IconButton(
+          iconSize: 10.0,
+          onPressed: mockOnPressedFunction,
+          icon: const Icon(Icons.link),
+          constraints: const BoxConstraints(),
+        ),
+      ),
+    );
+
+    final RenderBox iconButton = tester.renderObject(find.byType(IconButton));
+    expect(iconButton.size, const Size(26.0, 26.0));
```

Comment:
Please add a comment that explains where 26x26 comes from

Reply:
Done.